composer.json

```
@@ -62,7 +62,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.4.x-dev"
+            "dev-master": "1.5.x-dev"
```

```suggestion
            "dev-master": "1.6.x-dev"
```

Let's set `1.6.x-dev` as the next release branch.